protos/flyteidl/plugins/kubeflow/pytorch.proto

```diff
@@ -7,6 +7,16 @@ option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/plugins";
 import "flyteidl/core/tasks.proto";
 import "flyteidl/plugins/kubeflow/common.proto";
 
+// Custom proto for torch elastic config for distributed training using 
+// https://github.com/kubeflow/training-operator/blob/master/pkg/apis/kubeflow.org/v1/pytorch_types.go
+message ElasticConfig {
```

Comment:
FYI https://github.com/flyteorg/flyteidl/commit/6a7b31438821190be535cc6bbcb6f2939e5a056a

https://github.com/flyteorg/flyte/issues/3614

(Flytekit PR is not merged yet)

Reply:
@fg91 question, does elastic config require the master replica spec to be nil/not set?

Reply:
Unfortunately Kubeflow is very poorly documented but from [their examples](https://github.com/kubeflow/training-operator/blob/master/examples/pytorch/elastic/imagenet/imagenet.yaml) this is my understanding, yes.

I also handled it like this in [this](https://github.com/flyteorg/flyteplugins/pull/343/files) PR in `flyteplugins` and it works.